packages/kotti-ui/source/kotti-breadcrumb/KtBreadcrumb.vue

```diff
@@ -112,8 +118,12 @@ export default defineComponent<KottiBreadcrumb.PropsInternal>({
 			}
 		}
 
-		&-separator {
-			padding: 0 0.4rem;
+		&__separator {
+			display: flex;
+			align-items: center;
+			&__text {
+				padding: 0 0.4rem;
```

Comment:
Since this is only applied to the text class, breadcrumbs separated by icons will now have less spacing between them than breadcrumbs separated by chars.
The padding should go one level up to the `&__separator` section and you can remove `kt-breadcrumb__text__separator__text` class from the template